public/js/pimcore/object/tags/classificationstore.js

```diff
@@ -184,9 +169,28 @@ pimcore.object.tags.classificationstore = Class.create(pimcore.object.tags.abstr
                     hideMode: "offsets",
                     iconCls: icon,
                     title: title,
-                    items: childItems
-                });
+                    listeners: {
+                        activate: function(frontendLanguage, tab) {
+                            this.currentLanguage = frontendLanguage;
+
+                            const childItems = [];
+                            for (const groupId in this.fieldConfig.activeGroupDefinitions) {
+                                const groupedChildItems = [];
+
+                                if (this.fieldConfig.activeGroupDefinitions.hasOwnProperty(groupId)) {
+                                    const group = this.fieldConfig.activeGroupDefinitions[groupId];
+
+                                    const fieldset = this.createGroupFieldset(this.currentLanguage, group, groupedChildItems);
 
+                                    childItems.push(fieldset);
+                                }
+                            }
+
+                            tab.add(childItems);
```

Comment:
if i try to switch tab between the languages, eg. `EN`, `FR`, `EN`, `FR`, it would end up appending the items for each time i click on it.